bank.py

```diff
@@ -76,3 +76,23 @@ def save_users(self):
             for user in self.users:
                 f.write(f"{user.user_id}:{user.pin}:{user.balance}\n")
 
+    def change_user_pin(self, user: User):
+        """changes the user's pin"""
+        count_attempts_pin = 1
+        while count_attempts_pin <= 3:
```

Comment:
I can create the method that counts PIN_ATTEMPTS.  and I can use it in the main.py. But when attempts will be counted only for PIN while ID will be static. 
Maybe it will be better.